nixos/modules/services/backup/borgbackup.nix

```diff
@@ -277,6 +289,40 @@ in {
             example = "-u +%s";
           };
 
+          persistent = mkOption {
+            type = types.bool;
+            default = false;
+            description = ''
+              Whether backups should be triggered if they would have triggered
+              at least once while inactive
+            '';
+          };
+
+          randomizedDelaySec = mkOption {
+            type = types.str;
+            default = "0";
+            description = ''
+              Backups are delayed by a uniformly random amount of time up to
+              the value. Useful to stagger remote backups on a weak network
+              connection.
```

Comment:
maybe a reference to the used time format would be nice

Reply:
I repeated the reference to the man page from the `startAt` option for the new options that expect time spans and rebased. I guess I figured the other reference was close enough before but from a distance it is nicer the reference is right there, yeah.